services_app/src/androidTest/java/org/opendatakit/webkitserver/service/OdkWebserverServiceTest.java

```diff
@@ -195,7 +196,7 @@ public void testServingHelloWorldHtml() {
 
         IWebkitServerInterface serviceInterface = getIWebkitServerInterface();
 
-        PrintWriter writer = null;
```

Comment:
This change is actually making the code more confusing as it's an explicit initialization that can be used to clue the code reader into that we are expecting the writer to be null in the logic for a certain amount of time.

Reply:
Noted. I have pushed the changes please